update/mock/syncStateStub.go

```diff
@@ -3,15 +3,15 @@ package mock
 import (
 	"github.com/ElrondNetwork/elrond-go-core/data"
 	"github.com/ElrondNetwork/elrond-go-core/data/block"
-	"github.com/ElrondNetwork/elrond-go/state/temporary"
+	"github.com/ElrondNetwork/elrond-go/common"
```

Comment:
rename stubs and structs to `stateSyncer` instead of `syncState` ?

Reply:
done